### PR TITLE
[MLN-783] Changed Devise domain name to mylibrarynyc.org

### PR DESCRIPTION
**Pull request by gonuguntla**

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@mylibrarynyc.org'
```
Comment (thisisstephenbetts) [inline config/initializers/devise.rb:15]: Let's change that to 'passwordreset@mylibrarynyc.org'. Thanks Deepika!

**Review by thisisstephenbetts — CHANGES_REQUESTED** [review]
Thanks Deepika! I'd like the new password to be something more descriptive than "please-change-me...' I suggest using  'passwordreset@mylibrarynyc.org'. Thanks!

File: config/initializers/devise.rb (hunk 1 above)
Comment (gonuguntla) [reply config/initializers/devise.rb]: @thisisstephenbetts Done

**Review by thisisstephenbetts — APPROVED** [review]
Thank you!

**Pull request merged**